data-table/table.js

```diff
@@ -236,57 +258,115 @@ function cellIndexToElementId(wrapperDivId, rowIndex, colIndex) {
     return wrapperDivId + "row_" + rowIndex + "_and_col_" + colIndex + "_";
 }
 
-function deleteRows(numberOfRows, index) {
+/**
+ * Calls functions that create the two main entry boxes: one for rows, another for columns
+ * @param {object} config   - Table configuration object
+ * @returns {undefined}     - Doesn't return anything
+ */
+function createEntryBox(config) {
+    createRowEntryBox(config);
+
+    createColumnEntryBox();
+}
+
+/**
+ * Creates HTML elements by which the user can easily create rows with custom left-most cells.
+ * @param {object} config   - Table configuration object
+ * @returns {undefined}     - Doesn't return anything
+ */
+function createRowEntryBox(config) {
+    let entryBoxDiv = document.getElementById(config.tableIds.entryBoxDivId);
+
     /**
-     * Deletes row from existing table. Maybe should also
-     * accept index as an argument?
-     * TODO: Fill this out
+     * TODO: Implement (editable?) container for already existing row names
      */
-}
 
+    createRowInputAndBtn(config);
+
+    let br = document.createElement("br");
+    let br1 = document.createElement("br");
+    let br2 = document.createElement("br");
```

Comment:
Fine for now, but eventually replace: `<br>` should really only be used for breaking a line of text, and not for spacing. Spacing like this should be done with `margin`s and `padding`s in CSS. (Consider adding a TODO above this.)

Reply:
Added the TODO! (And you're totally right, it was a brute-force fix)